src/modules/trade/pure/ConfirmDetailsItem/index.tsx

```diff
@@ -2,23 +2,22 @@ import { ReactNode } from 'react'
 
 import { CornerDownRight } from 'react-feather'
 
+import { InfoIcon } from 'legacy/components/InfoIcon'
 import { RowFixed } from 'legacy/components/Row'
-import { MouseoverTooltipContent } from 'legacy/components/Tooltip'
 
 import { TextWrapper } from 'modules/swap/pure/Row/styled'
-import { StyledInfoIcon } from 'modules/swap/pure/styled'
 
 import { Content, Row, Wrapper } from './styled'
 
-type Props = {
+export type ConfirmDetailsItemProps = {
```

Comment:
thanks, i probably didn't justify why I mentioned, but one reason is that is convenient to use when defining cosmos fixtures. Also, might be handy for some other things, and doesn't have a cost to add it (its just a type)